bot/main.py

```diff
@@ -50,7 +50,7 @@ def get_config(server_id: str):
             return section_obj
 
     print(f"Server id {server_id} not found in server config")
-    exit(1)
+    return None
```

Comment:
All functions that call this code must now handle the `None` return case, so that they don't error at runtime.

In the `post_verification` function, I'd like to have a message sent on the discord channel saying that the bot hasn't been configured to work with this channel.

In the `is_academic` function, I'd like it to return `False` in this case.